audio/engine.asm

```diff
@@ -1446,9 +1470,14 @@ Music_Ret:
 	ld [hl], d
 	ret
 
-Music_Call:
 ; call music stream (subroutine)
-; parameters: ll hh ; pointer to subroutine
+; input: 
+	; bc = start of current channel
+; output: 
+	; CHANNEL_LAST_MUSIC_ADDRESS = CHANNEL_MUSIC_ADDRESS
+	; CHANNEL_MUSIC_ADDRESS = pointer to subroutine (ll hh)
+	; Sets the SOUND_SUBROUTINE flag in CHANNEL_FLAGS1 
```

Comment:
`CHANNEL_LAST_MUSIC_ADDRESS` and `CHANNEL_MUSIC_ADDRESS` are constants, and thus, they cannot ever be outputs in the first place. In fact, all inputs and outputs must be in a register or a memory location, and those are neither. Are you sure you're understanding the function in the first place?

Also, this excess verbosity in function comments isn't helpful; we're not trying to run doxygen on the codebase. The files are already massive as it is; keep your comments succint and to the point.